src/SHAutomation.Core/AutomationElements/SHAutomationElement.FindAllDescendantsNameContains.cs

```diff
@@ -18,7 +18,7 @@ public ISHAutomationElement[] FindAllDescendantsNameContains(string name, TimeSp
             bool getElements(bool shouldExist)
```

Comment:
If we're changing this could we change `getElements` to be a private method outside of this one?